components/event-feed-service/pkg/persistence/mappings.go

```diff
@@ -144,6 +144,14 @@ var feedProps = `
 		},
 		"projects": {
 			"type": "keyword"
+		},
+		"parent_name": {
+			"type": "keyword",
+			"ignore_above": 256
+		},
+		"parent_type": {
+			"type": "keyword",
+			"ignore_above": 256
```

Comment:
Chef Actions have a parent type and parent name to allow a child-parent relationship.